pkg/result/processors/autogenerated_exclude.go

```diff
@@ -40,11 +51,7 @@ func (p *AutogeneratedExclude) Process(issues []result.Issue) ([]result.Issue, e
 	return filterIssuesErr(issues, p.shouldPassIssue)
 }
 
-func isSpecialAutogeneratedFile(filePath string) bool {
-	fileName := filepath.Base(filePath)
-	// fake files or generation definitions to which //line points to for generated files
-	return filepath.Ext(fileName) != ".go"
-}
+func (p *AutogeneratedExclude) Finish() {}
 
 func (p *AutogeneratedExclude) shouldPassIssue(issue *result.Issue) (bool, error) {
```

Comment:
Perhaps we should cover this function with some tests due to the new `strict` logic.

Or even better, we should add tests for the exported `AutogeneratedExclude.Process` function.